client/src/app.tsx

```diff
@@ -111,7 +111,7 @@ function App() {
               </div>
             ) : (
               <>
-                {connection == null && !sessionMetadata.isGameEnd && (
+                {connection == null && !sessionMetadata.isGameEnd && !roomIdFromUrl && (
```

Comment:
Added this so lobby doesn't flash when we already know player is trying to join a game. Instead just flashes a blank screen which leads into loading screen nicely